src/Components/Label/index.css

```diff
@@ -0,0 +1,14 @@
+.labelTitle {
```

Comment:
This is not following `SUIT CSS` naming conventions. Please take another look at the documentation [here](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md)